src/QMCWaveFunctions/Fermion/SlaterDetOpt.h

```diff
@@ -181,11 +181,11 @@ class SlaterDetOpt : public DiracDeterminantBase {
 
     void restore(int iat);
 
-    RealType registerData(ParticleSet& P, BufferType& buf);
+    void registerData(ParticleSet& P, WFBufferType& buf);
```

Comment:
The return type is changed here also. Please explain.

Reply:
registerData is no more responsible for the initial computing. The return value is no more valid.
See OrbitalBase change in #521.